### ISSUE TITLE
Could not create the target notebook in Onenote!

### ISSUE THREAD

**Issue by timlhicks** [stefankueng/EvImSync#42]
When I run evernote2onenote, I select an enex file, and I get the attached complete error. Some notes imply you need a certain version of OneNote. I am running on windows 10 and I have the application installed and also Office365. What do I need to do? Please Help.

<img width="296" alt="Evernote2OneNoteErrorScreenshot 2023-10-12 153910" src="https://github.com/stefankueng/EvImSync/assets/7827284/5973ae45-fca7-41e8-95b5-bb885eb07d3c">

Tim
timlhicks@gmail.com
506-269-7112

**Comment by John-W1JA** [comment]
I have the same issue, and get the same error message. Exactly the same, right down to the ...IID '{452AC71A-B655-4967-A208-A4CC39DD7949}' part, for example.

**Comment by PaulMpls** [comment]
Same issue for me
![image](https://github.com/stefankueng/EvImSync/assets/145939625/c3cc5c55-89b1-4496-a580-ad19bbd8efb6)

**Comment by PaulMpls** [comment]
This looks like a duplicate of https://github.com/stefankueng/EvImSync/issues/30

**Comment by peters248** [comment]
For what it's worth, I'm getting the identical error message , even after installing the latest Evernote maintenance upgrade last week.

**Comment by stefankueng** [comment]
the target notebook name is taken from the enex file name. Make sure the filename does not have any special chars in it.

**Comment by peters248** [comment]
The target notebook name and the Evernote notebook have the same 
one-word file name comprising only letters: no digits or other 
characters of any type.



------ Original Message ------
From "Stefan Küng" ***@***.***>
To "stefankueng/EvImSync" ***@***.***>
Cc "peters248" ***@***.***>; "Comment" 
***@***.***>
Date 11/30/2023 10:16:28 AM
Subject Re: [stefankueng/EvImSync] Could not create the target notebook 
in Onenote! (Issue #42)

**Comment by standupguru** [comment]
I have this exact same issue.  Same exact error message.  Using OneNote 2016.  And my Evernote notebooks do have very short names with no special characters.  The names are things like Blog, Books, Fitness, etc.  

Please help.  I am eager to get everything into OneNote. 

Thanks!!!

**Comment by ullib61** [comment]
same issue here with exact the same error message like above. Notebook name and filename contain only alphanumeric characters (e.g. Misc).
Any hints or hope to get this solved?
Thanks

**Comment by SirSparkles** [comment]
See #65 too

**Comment by ullib61** [comment]
On my side I worked around the issue by doing the convertion on a Win10 machine. That's working fine with the same enex files that fail on the Win11 machine.

**Comment by kervinpierre** [comment]
In my case, this seems to relate to stacks.  Notebooks not in a stack import.  Those in a stack produce this error.

**Comment by yihchuang** [comment]
After I repaired Microsoft 365 - en-us (completely via "Online Repair" option) on Windows 10, I then successfully imported the .ENEX files from Evernote to OneNote.

![image](https://github.com/stefankueng/EvImSync/assets/8800135/ee3cc50b-bbdc-4a22-b61f-c1c3affcc72e)

**Comment by peters248** [comment]
Thanks for the suggestion.  It worked with one of my notebooks, although 
only after three attempts.  However, with a much larger notebook, 
Evernote keps stopping at various notes that blocked the export, telling 
me to fix an undefined problem and, then, restarting the whole process 
to export to an ENEX file. This is entirely an Evernote problem (which I 
have absolutely no confidence they'll fix).



------ Original Message ------
From "yihchuang" ***@***.***>
To "stefankueng/EvImSync" ***@***.***>
Cc "peters248" ***@***.***>; "Comment" 
***@***.***>
Date 4/10/2024 3:57:31 PM
Subject Re: [stefankueng/EvImSync] Could not create the target notebook 
in Onenote! (Issue #42)

**Comment by yihchuang** [comment]
I successfully imported three .ENEX files, and the largest file size was
74MB (including 3657 notes).  When I ran the Evernote2Onenote application (version 1.3.6.241),
I managed to keep my Windows 10 (16GB RAM) memory usage below 50%.  Hope this helps.
